spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    let!(:books) { create_list(:book, 3) }
+
+    before do
+      get :index
+    end
+
+    it { expect(response).to have_http_status(:ok) }
+
+    it 'responds with the books list in json format' do
+      expected = ActiveModel::Serializer::CollectionSerializer.new(
+        books, each_serializer: BookSerializer, root: false
+      ).to_json
+      expect(response.body.to_json) =~ JSON.parse(expected)
```

Comment:
In the example the test says `response_body`. We usually have this helper for tests to parse the response body.
```
module Response
  module JSONParser
    def response_body
      ActiveSupport::JSON.decode(response.body) if response.present? && response.body.present?
    end
  end
end
```
Add it in **spec/support**

Reply:
Also add  `config.include Response::JSONParser, type: :controller` to your rails_helper

Reply:
@facucompe Done!